### ISSUE TITLE
[Feature Request] Make Hammer++ available for GMod/Increase the compiler limits

### ISSUE THREAD

**Issue by Millar15** [ficool2/HammerPlusPlus-Issue-Tracker#29]
- Make it so Hammer++ can be used with GMod compilers 

OR

- Increase the compiler limits for Source SDK 2013, such as max brushes, max brush sides, entities etc etc

**Comment by ficool2** [comment]
Hammer++ can already use GMod compilers, just modify your game configuration and use the compilers in GMod's bin folder

**Comment by Millar15** [comment]
I'll try it again but last time it didn't work.

**Comment by MARK2580** [comment]
> I'll try it again but last time it didn't work (will update you accordingly).

Personally, I just hardlinked the addons and custom folders. And I compile maps using the cmd line, because in hammer++ compilation settings (were) broken. https://github.com/ficool2/HammerPlusPlus-Issue-Tracker/issues/33 I also included the truncated content from gmod to the SDK 2013 so that all the textures and models used in the sandbox were available (bypassing the dev textures that break hammer++).

**Comment by Millar15** [comment]
Yeah, it worked but unfortunately I can't use Slammin' Tools with the GMod compilers :(
